### PR TITLE
[enhancement] (odbc) support more ODBC Connection Parameters for odbc external table

### PR DESCRIPTION
**Pull request by yangzhg**
# Proposed changes

user can add more supported ODBC Connection Parameters to resoure PROPERTIES or exteranl table PROPERTIES

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/faq/error.md [hunk 1]
```diff
@@ -149,3 +149,16 @@ failed to initialize storage reader. tablet=63416.1050661139.aa4d304e7a7aff9c-f0
 ### E12.Doris编译安装JDK版本不兼容问题
 
 在自己使用 Docker 编译 Doris 的时候，编译完成安装以后启动FE，出现 ```java.lang.Suchmethoderror: java.nio. ByteBuffer. limit (I)Ljava/nio/ByteBuffer;``` 异常信息，这是因为Docker里默认是JDK 11，如果你的安装环境是使用JDK8 ，需要在 Docker 里 JDK 环境切换成 JDK8，具体切换方法参照[编译](https://doris.apache.org/zh-CN/installing/compilation.html)
+
+### E13. Doris 升级到1.0 以后版本通过ODBC访问MySQL外表报错 `Failed to set ciphers to use (2026)`
+这个问题出现在doris 升级到1.0 版本以后，且使用 Connector/ODBC 8.0.x 以上版本，Connector/ODBC 8.0.x 有多重获取方式，比如通过yum安装的的方式获取的 `/usr/lib64/libmyodbc8w.so` 依赖的是 `libssl.so.10` 和 `libcrypto.so.10`
```
Comment (morningman) [inline docs/zh-CN/faq/error.md:154]:
```suggestion
这个问题出现在doris 升级到1.0 版本以后，且使用 Connector/ODBC 8.0.x 以上版本，Connector/ODBC 8.0.x 有多种获取方式，比如通过yum安装的的方式获取的 `/usr/lib64/libmyodbc8w.so` 依赖的是 `libssl.so.10` 和 `libcrypto.so.10`
```

File: docs/zh-CN/faq/error.md [hunk 2]
```diff
@@ -149,3 +149,16 @@ failed to initialize storage reader. tablet=63416.1050661139.aa4d304e7a7aff9c-f0
...
+而doris 1.0 以后版本中openssl 已经升级到1.1 且内置在doris 二进制包中，因此会导致 openssl 的冲突进而出现 类似 如下的错误
+```
+ERROR 1105 (HY000): errCode = 2, detailMessage = driver connect Error: HY000 [MySQL][ODBC 8.0(w) Driver]SSL connection error: Failed to set ciphers to use (2026)
+```
+解决方式是使用`Connector/ODBC 8.0.28` 版本的 ODBC Connector， 并且选择 在操作系统处选择 `Linux - Generic`, 这个版本的ODBC Driver 使用 openssl 1，1 版本。具体使用方式见 [ODBC外表使用文档](../extending-doris/odbc-of-doris.md)
```
[the remark this answers is not in the document]
Comment (morningman) [reply docs/zh-CN/faq/error.md:159]: 这里直接给一个mysql driver的下载连接吧

[the remark this answers is not in the document]
Comment (yangzhg) [reply docs/zh-CN/faq/error.md]: mysql的下载地址再发布新版本后会变化

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**